src/server/utils/devCdnFactory.js

```diff
@@ -33,9 +33,15 @@ const getLocalModuleMap = ({ pathToModuleMap, oneAppDevCdnAddress }) => {
   const moduleMap = JSON.parse(fs.readFileSync(pathToModuleMap, 'utf8').toString());
   Object.keys(moduleMap.modules).forEach((moduleName) => {
     const module = moduleMap.modules[moduleName];
-    module.browser.url = module.browser.url.replace('[one-app-dev-cdn-url]', oneAppDevCdnAddress);
-    module.legacyBrowser.url = module.legacyBrowser.url.replace('[one-app-dev-cdn-url]', oneAppDevCdnAddress);
-    module.node.url = module.node.url.replace('[one-app-dev-cdn-url]', oneAppDevCdnAddress);
+    Object
+      .values(module)
+      .filter((bundle) => Object.hasOwnProperty.call(bundle, 'url'))
```

Comment:
this is defensive; do we need to be? e.g. is `baseUrl` always computed or can it be part of the module map JSON resource?